libs/server.js

```diff
@@ -126,6 +125,7 @@ function startServer(){
 	naming.configure(miaou);
 
 	var cookieParser = require('cookie-parser')(miaou.config.secret);
+	var sessionStore = new RedisStore(miaou.config.redisStore);
```

Comment:
I thought that was unnecessary because this check is apply anywhere else (look at the line just above)